### ISSUE TITLE
Model estimation: vehicle choice model

### ISSUE THREAD

**Issue by ColSmith76** [SEMCOG/SEMCOG_CVTM#37]
Transferred model includes an estimated vehicle choice model based on the Ohio GES. Update using the SEMCOG CVS data

**Comment by ColSmith76** [comment]
@jeffdumont  this is a good one to start with. I think.  I'll some more comments and info this model and what we need to do. Can discuss tomorrow.

**Comment by jeffdumont** [comment]
@ColSmith76 

I got to a similar specification using Apollo. Data prep and estimation code has been pushed. I did group some of the industries and activities different due to data sparsity so the specification is similar but not identical.

Some questions/comments:
* We have some data on vehicle availability in the CVS (e.g., vehicle owned/leased by establishment). Should we incorporate this into the model? The previous model assumed all 3 were available. Doing so drops the fit substantially.
* I used site_id to incorporate a panel effect. The robust t-ratios are adjusted downward to represent this effect.
* Should I proceed with a specification search?

```
LL(start)                        : -11487.09
LL(0)                            : -11487.09
LL(final)                        : -7716.255
Rho-square (0)                   :  0.3283 
Adj.Rho-square (0)               :  0.3255 
AIC                              :  15496.51 
BIC                              :  15728.67 
Estimated parameters             :  32
Time taken (hh:mm:ss)            :  00:01:42.69 
Iterations                       :  43 
Min abs eigenvalue of hessian    :  4.932299 

Estimates:
                              Estimate Std.err. t.ratio(0) Rob.std.err. Rob.t.ratio(0)
asc_light                       2.2944   0.1271      18.05       0.4434           5.18
asc_medium                      0.5217   0.1489       3.50       0.5004           1.04
asc_heavy                       0.0000       NA         NA           NA             NA
beta_v1_industry_cons          -0.8413   0.1359      -6.19       0.5296          -1.59
beta_v1_industry_manu          -0.4705   0.1508      -3.12       0.6672          -0.71
beta_v1_industry_whol           0.0000       NA         NA           NA             NA
beta_v1_industry_reta          -0.1215   0.1504      -0.81       0.5591          -0.22
beta_v1_industry_tran          -2.4438   0.1236     -19.77       0.5439          -4.49
beta_v1_industry_serv           1.3259   0.2023       6.55       0.6547           2.03
beta_v1_industry_othe          -2.3191   0.1792     -12.94       0.8091          -2.87
beta_v2_industry_cons          -0.0483   0.1573      -0.31       0.6104          -0.08
beta_v2_industry_manu          -0.5418   0.1855      -2.92       0.6887          -0.79
beta_v2_industry_whol           0.0000       NA         NA           NA             NA
beta_v2_industry_reta          -0.4478   0.1846      -2.43       0.6241          -0.72
beta_v2_industry_tran          -1.3913   0.1443      -9.64       0.5851          -2.38
beta_v2_industry_serv           1.8962   0.2169       8.74       0.7631           2.48
beta_v2_industry_othe          -0.2281   0.1856      -1.23       0.9733          -0.23
beta_v1_activity_return         0.0000       NA         NA           NA             NA
beta_v1_activity_driver_needs   1.2655   0.1474       8.59       0.1976           6.40
beta_v1_activity_deliver       -1.2410   0.0875     -14.19       0.2364          -5.25
beta_v1_activity_pickup        -1.3716   0.1121     -12.23       0.5576          -2.46
beta_v1_activity_service        1.8173   0.1442      12.60       0.3967           4.58
beta_v1_activity_other          1.5777   0.1309      12.06       0.2538           6.22
beta_v2_activity_return         0.0000       NA         NA           NA             NA
beta_v2_activity_driver_needs   0.1321   0.1857       0.71       0.2864           0.46
beta_v2_activity_deliver       -0.1475   0.0971      -1.52       0.2946          -0.50
beta_v2_activity_pickup        -0.3277   0.1200      -2.73       0.2513          -1.30
beta_v2_activity_service        2.1752   0.1511      14.39       0.3936           5.53
beta_v2_activity_other          0.9877   0.1488       6.64       0.3325           2.97
beta_v1_dist_00_02              0.0000       NA         NA           NA             NA
beta_v1_dist_02_05              0.5666   0.1142       4.96       0.1580           3.59
beta_v1_dist_05_10              0.3042   0.1091       2.79       0.1444           2.11
beta_v1_dist_10_20             -0.0627   0.0987      -0.64       0.2000          -0.31
beta_v1_dist_20_p              -0.8864   0.0871     -10.18       0.1546          -5.73
beta_v2_dist_00_02              0.0000       NA         NA           NA             NA
beta_v2_dist_02_05              0.3703   0.1231       3.01       0.1770           2.09
beta_v2_dist_05_10              0.2591   0.1173       2.21       0.1661           1.56
beta_v2_dist_10_20             -0.3439   0.1104      -3.11       0.2145          -1.60
beta_v2_dist_20_p              -1.0765   0.1010     -10.66       0.1839          -5.85

Overview of choices for model component "MNL"
                                    light   medium   heavy
Times available                  10456.00 10456.00 10456.0
Times chosen                      6359.00  2048.00  2049.0
Percentage chosen overall           60.82    19.59    19.6
Percentage chosen when available    60.82    19.59    19.6

```

**Comment by ColSmith76** [comment]
@jeffdumont thanks, good start.

A few comments: from the data prep code, it looks like you've include vehicle categories 6 and 7 in heavy (choice 3).

# 1=Passenger Car
# 2=Pick-up Truck (4 wheels)
# 3=Van (Cargo or Minivan) (4 wheels)
# 4=Sport Utility Vehicle (SUV) (4 wheels)
# 5=Single Unit 2-axle (6 wheels)
# 6=Single Unit 3-axle (10 wheels)
# 7=Single Unit 4-axle (14 wheels)
# 8=Semi (all Tractor-Trailer combinations)
# 888=Other

choice_data[, .N, VEH_CLASS][order(VEH_CLASS)]

choice_data[VEH_CLASS %in% 1:4, veh_choice := 1]
choice_data[VEH_CLASS %in% 5, veh_choice := 2]
choice_data[VEH_CLASS %in% c(6:8, 888), veh_choice := 3]

Can you group VEH_CLASS 5, 6, 7 into vehicle choice 2 --- that is all medium trucks.
Then vehicle choice 3 would be VEH_CLASS 8.
Does including or excluding the 888 (others) make much difference?

**Comment by ColSmith76** [comment]
@jeffdumont please could you also consider the Model_EmpCat groupings for the establishment industry in the model? We are using those in application as opposed to the slightly simple industrial classification used in the survey. The establishments table should already have the Model_EmpCat field added.

**Comment by ColSmith76** [comment]
@jeffdumont  so far we've got vehicle choice as a function of 
-- establishment industry
-- stop activity
-- stop distance from establishment [is that how distance is defined or are you using trip distance?]

Some thoughts:
1. The modal as the point of application knows distance of the stop to the establishment but not any relationship to other stops. So distance needs to be establishment to stop distance (either network or straightline) -- if that is not in the survey that we need to grab it from the skims.
2. This model is particularly looking at vehicle choice for scheduled stops. That would only be stops for pickup, drop, and service. We'll deal with driver needs, other, return stops in other ways.

**Comment by jeffdumont** [comment]
Here is an updated model. Code has been pushed. 

Changes include:
* modeling only stops for delivery, pickup and service
* changed industry effects to be based on empcat
* reclassified all single axle trucks to be medium
* calculated straightline distance based on TAZ/ZIP code centroids.
  * used the same bins as before
* removed 2 establishments with external SITE_TAZs
* removed external trips with funky ZIP Codes (non-identifiable)

```
Model name                       : Match Previous Estimation
Model description                : MNL model - previous specification
Model run at                     : 2020-08-26 12:03:12
Estimation method                : bfgs
Model diagnosis                  : successful convergence 
Number of individuals            : 623
Number of observations           : 4595

Number of cores used             :  1 
Model without mixing

LL(start)                        : -5048.123
LL(0)                            : -5048.123
LL(final)                        : -3575.14
Rho-square (0)                   :  0.2918 
Adj.Rho-square (0)               :  0.2866 
AIC                              :  7202.28 
BIC                              :  7369.53 
Estimated parameters             :  26
Time taken (hh:mm:ss)            :  00:00:53.76 
Iterations                       :  40 
Min abs eigenvalue of hessian    :  1.440761 

Estimates:
                         Estimate Std.err. t.ratio(0) Rob.std.err. Rob.t.ratio(0)
asc_light                  2.7197   0.2523      10.78       0.6429           4.23
asc_medium                 1.5590   0.2659       5.86       0.6639           2.35
asc_heavy                  0.0000       NA         NA           NA             NA
beta_v1_industry_cons     -3.1816   0.2608     -12.20       0.7385          -4.31
beta_v1_industry_manu     -1.2995   0.2742      -4.74       0.7923          -1.64
beta_v1_industry_whol      0.0000       NA         NA           NA             NA
beta_v1_industry_reta     -1.1184   0.2593      -4.31       0.7087          -1.58
beta_v1_industry_tran     -3.6082   0.2267     -15.92       0.6819          -5.29
beta_v1_industry_serv     -0.4542   0.3173      -1.43       0.8631          -0.53
beta_v1_industry_othe     -5.7859   0.5422     -10.67       1.0881          -5.32
beta_v2_industry_cons     -1.8967   0.2658      -7.14       0.7067          -2.68
beta_v2_industry_manu     -0.9553   0.2896      -3.30       0.6621          -1.44
beta_v2_industry_whol      0.0000       NA         NA           NA             NA
beta_v2_industry_reta     -1.5189   0.2857      -5.32       0.6722          -2.26
beta_v2_industry_tran     -2.5600   0.2345     -10.92       0.6357          -4.03
beta_v2_industry_serv      0.5566   0.3231       1.72       0.9366           0.59
beta_v2_industry_othe     -3.3235   0.3758      -8.84       1.1033          -3.01
beta_v1_activity_deliver   0.0000       NA         NA           NA             NA
beta_v1_activity_pickup   -0.0699   0.1276      -0.55       0.3861          -0.18
beta_v1_activity_service   3.6713   0.1834      20.02       0.5139           7.14
beta_v2_activity_deliver   0.0000       NA         NA           NA             NA
beta_v2_activity_pickup   -0.1715   0.1238      -1.39       0.1987          -0.86
beta_v2_activity_service   2.5085   0.1833      13.68       0.5165           4.86
beta_v1_dist_00_02         0.0000       NA         NA           NA             NA
beta_v1_dist_02_05         0.3373   0.1964       1.72       0.2983           1.13
beta_v1_dist_05_10         0.3674   0.1805       2.04       0.4846           0.76
beta_v1_dist_10_20        -0.6463   0.1722      -3.75       0.3750          -1.72
beta_v1_dist_20_p         -1.3614   0.1680      -8.10       0.3765          -3.62
beta_v2_dist_00_02         0.0000       NA         NA           NA             NA
beta_v2_dist_02_05         0.8218   0.2025       4.06       0.3806           2.16
beta_v2_dist_05_10         0.9409   0.1851       5.08       0.5282           1.78
beta_v2_dist_10_20        -0.1131   0.1784      -0.63       0.4585          -0.25
beta_v2_dist_20_p         -0.9715   0.1761      -5.52       0.4605          -2.11

Overview of choices for model component "MNL"
                                   light  medium   heavy
Times available                  4595.00 4595.00 4595.00
Times chosen                     2234.00 1283.00 1078.00
Percentage chosen overall          48.62   27.92   23.46
Percentage chosen when available   48.62   27.92   23.46
```

**Comment by ColSmith76** [comment]
@jeffdumont I think that is a good first version of the model to try out in application, the parameters largely make sense in terms of direction and order.

**Comment by JilanChen** [comment]
> @ColSmith76
> 
> I got to a similar specification using Apollo. Data prep and estimation code has been pushed. I did group some of the industries and activities different due to data sparsity so the specification is similar but not identical.
> 
> Some questions/comments:
> 
> * We have some data on vehicle availability in the CVS (e.g., vehicle owned/leased by establishment). Should we incorporate this into the model? The previous model assumed all 3 were available. Doing so drops the fit substantially.
> * I used site_id to incorporate a panel effect. The robust t-ratios are adjusted downward to represent this effect.
> * Should I proceed with a specification search?
> 
> ```
> LL(start)                        : -11487.09
> LL(0)                            : -11487.09
> LL(final)                        : -7716.255
> Rho-square (0)                   :  0.3283 
> Adj.Rho-square (0)               :  0.3255 
> AIC                              :  15496.51 
> BIC                              :  15728.67 
> Estimated parameters             :  32
> Time taken (hh:mm:ss)            :  00:01:42.69 
> Iterations                       :  43 
> Min abs eigenvalue of hessian    :  4.932299 
> 
> Estimates:
>                               Estimate Std.err. t.ratio(0) Rob.std.err. Rob.t.ratio(0)
> asc_light                       2.2944   0.1271      18.05       0.4434           5.18
> asc_medium                      0.5217   0.1489       3.50       0.5004           1.04
> asc_heavy                       0.0000       NA         NA           NA             NA
> beta_v1_industry_cons          -0.8413   0.1359      -6.19       0.5296          -1.59
> beta_v1_industry_manu          -0.4705   0.1508      -3.12       0.6672          -0.71
> beta_v1_industry_whol           0.0000       NA         NA           NA             NA
> beta_v1_industry_reta          -0.1215   0.1504      -0.81       0.5591          -0.22
> beta_v1_industry_tran          -2.4438   0.1236     -19.77       0.5439          -4.49
> beta_v1_industry_serv           1.3259   0.2023       6.55       0.6547           2.03
> beta_v1_industry_othe          -2.3191   0.1792     -12.94       0.8091          -2.87
> beta_v2_industry_cons          -0.0483   0.1573      -0.31       0.6104          -0.08
> beta_v2_industry_manu          -0.5418   0.1855      -2.92       0.6887          -0.79
> beta_v2_industry_whol           0.0000       NA         NA           NA             NA
> beta_v2_industry_reta          -0.4478   0.1846      -2.43       0.6241          -0.72
> beta_v2_industry_tran          -1.3913   0.1443      -9.64       0.5851          -2.38
> beta_v2_industry_serv           1.8962   0.2169       8.74       0.7631           2.48
> beta_v2_industry_othe          -0.2281   0.1856      -1.23       0.9733          -0.23
> beta_v1_activity_return         0.0000       NA         NA           NA             NA
> beta_v1_activity_driver_needs   1.2655   0.1474       8.59       0.1976           6.40
> beta_v1_activity_deliver       -1.2410   0.0875     -14.19       0.2364          -5.25
> beta_v1_activity_pickup        -1.3716   0.1121     -12.23       0.5576          -2.46
> beta_v1_activity_service        1.8173   0.1442      12.60       0.3967           4.58
> beta_v1_activity_other          1.5777   0.1309      12.06       0.2538           6.22
> beta_v2_activity_return         0.0000       NA         NA           NA             NA
> beta_v2_activity_driver_needs   0.1321   0.1857       0.71       0.2864           0.46
> beta_v2_activity_deliver       -0.1475   0.0971      -1.52       0.2946          -0.50
> beta_v2_activity_pickup        -0.3277   0.1200      -2.73       0.2513          -1.30
> beta_v2_activity_service        2.1752   0.1511      14.39       0.3936           5.53
> beta_v2_activity_other          0.9877   0.1488       6.64       0.3325           2.97
> beta_v1_dist_00_02              0.0000       NA         NA           NA             NA
> beta_v1_dist_02_05              0.5666   0.1142       4.96       0.1580           3.59
> beta_v1_dist_05_10              0.3042   0.1091       2.79       0.1444           2.11
> beta_v1_dist_10_20             -0.0627   0.0987      -0.64       0.2000          -0.31
> beta_v1_dist_20_p              -0.8864   0.0871     -10.18       0.1546          -5.73
> beta_v2_dist_00_02              0.0000       NA         NA           NA             NA
> beta_v2_dist_02_05              0.3703   0.1231       3.01       0.1770           2.09
> beta_v2_dist_05_10              0.2591   0.1173       2.21       0.1661           1.56
> beta_v2_dist_10_20             -0.3439   0.1104      -3.11       0.2145          -1.60
> beta_v2_dist_20_p              -1.0765   0.1010     -10.66       0.1839          -5.85
> 
> Overview of choices for model component "MNL"
>                                     light   medium   heavy
> Times available                  10456.00 10456.00 10456.0
> Times chosen                      6359.00  2048.00  2049.0
> Percentage chosen overall           60.82    19.59    19.6
> Percentage chosen when available    60.82    19.59    19.6
> ```

All vehicle types (owned or not owned) should be considered as available, right?

**Comment by JilanChen** [comment]
> Here is an updated model. Code has been pushed.
> 
> Changes include:
> 
> * modeling only stops for delivery, pickup and service
> * changed industry effects to be based on empcat
> * reclassified all single axle trucks to be medium
> * calculated straightline distance based on TAZ/ZIP code centroids.
>   
>   * used the same bins as before
> * removed 2 establishments with external SITE_TAZs
> * removed external trips with funky ZIP Codes (non-identifiable)
> 
> ```
> Model name                       : Match Previous Estimation
> Model description                : MNL model - previous specification
> Model run at                     : 2020-08-26 12:03:12
> Estimation method                : bfgs
> Model diagnosis                  : successful convergence 
> Number of individuals            : 623
> Number of observations           : 4595
> 
> Number of cores used             :  1 
> Model without mixing
> 
> LL(start)                        : -5048.123
> LL(0)                            : -5048.123
> LL(final)                        : -3575.14
> Rho-square (0)                   :  0.2918 
> Adj.Rho-square (0)               :  0.2866 
> AIC                              :  7202.28 
> BIC                              :  7369.53 
> Estimated parameters             :  26
> Time taken (hh:mm:ss)            :  00:00:53.76 
> Iterations                       :  40 
> Min abs eigenvalue of hessian    :  1.440761 
> 
> Estimates:
>                          Estimate Std.err. t.ratio(0) Rob.std.err. Rob.t.ratio(0)
> asc_light                  2.7197   0.2523      10.78       0.6429           4.23
> asc_medium                 1.5590   0.2659       5.86       0.6639           2.35
> asc_heavy                  0.0000       NA         NA           NA             NA
> beta_v1_industry_cons     -3.1816   0.2608     -12.20       0.7385          -4.31
> beta_v1_industry_manu     -1.2995   0.2742      -4.74       0.7923          -1.64
> beta_v1_industry_whol      0.0000       NA         NA           NA             NA
> beta_v1_industry_reta     -1.1184   0.2593      -4.31       0.7087          -1.58
> beta_v1_industry_tran     -3.6082   0.2267     -15.92       0.6819          -5.29
> beta_v1_industry_serv     -0.4542   0.3173      -1.43       0.8631          -0.53
> beta_v1_industry_othe     -5.7859   0.5422     -10.67       1.0881          -5.32
> beta_v2_industry_cons     -1.8967   0.2658      -7.14       0.7067          -2.68
> beta_v2_industry_manu     -0.9553   0.2896      -3.30       0.6621          -1.44
> beta_v2_industry_whol      0.0000       NA         NA           NA             NA
> beta_v2_industry_reta     -1.5189   0.2857      -5.32       0.6722          -2.26
> beta_v2_industry_tran     -2.5600   0.2345     -10.92       0.6357          -4.03
> beta_v2_industry_serv      0.5566   0.3231       1.72       0.9366           0.59
> beta_v2_industry_othe     -3.3235   0.3758      -8.84       1.1033          -3.01
> beta_v1_activity_deliver   0.0000       NA         NA           NA             NA
> beta_v1_activity_pickup   -0.0699   0.1276      -0.55       0.3861          -0.18
> beta_v1_activity_service   3.6713   0.1834      20.02       0.5139           7.14
> beta_v2_activity_deliver   0.0000       NA         NA           NA             NA
> beta_v2_activity_pickup   -0.1715   0.1238      -1.39       0.1987          -0.86
> beta_v2_activity_service   2.5085   0.1833      13.68       0.5165           4.86
> beta_v1_dist_00_02         0.0000       NA         NA           NA             NA
> beta_v1_dist_02_05         0.3373   0.1964       1.72       0.2983           1.13
> beta_v1_dist_05_10         0.3674   0.1805       2.04       0.4846           0.76
> beta_v1_dist_10_20        -0.6463   0.1722      -3.75       0.3750          -1.72
> beta_v1_dist_20_p         -1.3614   0.1680      -8.10       0.3765          -3.62
> beta_v2_dist_00_02         0.0000       NA         NA           NA             NA
> beta_v2_dist_02_05         0.8218   0.2025       4.06       0.3806           2.16
> beta_v2_dist_05_10         0.9409   0.1851       5.08       0.5282           1.78
> beta_v2_dist_10_20        -0.1131   0.1784      -0.63       0.4585          -0.25
> beta_v2_dist_20_p         -0.9715   0.1761      -5.52       0.4605          -2.11
> 
> Overview of choices for model component "MNL"
>                                    light  medium   heavy
> Times available                  4595.00 4595.00 4595.00
> Times chosen                     2234.00 1283.00 1078.00
> Percentage chosen overall          48.62   27.92   23.46
> Percentage chosen when available   48.62   27.92   23.46
> ```

What is the reason we use the straight line instead of grabbing the distance from the skims?

**Comment by jeffdumont** [comment]
estimation_data.rds and cv_vehicle_model.rds have been pushed @ColSmith76 .

**Comment by ColSmith76** [comment]
Re-estimate using corrected skims as opposed to centroid to centroid distances. 

Check the "other" industry and what the survey data shows there

Then we can document and test in application

**Comment by jeffdumont** [comment]
The "Other" industry is primarily classified as Natural Resources and Mining so not surprising there is a strong preference for heavy trucks as expressed by negative coefficients on the light and medium vehicle alternatives.

**Comment by ColSmith76** [comment]
@jeffdumont specification changes for the vehicle choice model:
- update employment categories to match/nest within the set developed by @goreaditya for the activities and stops model (moderately different than the groupings currently being used), main difference is dealing with the "other" group
- at the point that this model is applied we know whether a stop is goods or service but do not split between pickup and delivery, so please can you combine the pickup and delivery specific variables into a single goods stop variable? (they aren't significantly different from each other anyway so this won't have much impact on the model)

otherwise, looks good.

**Comment by jeffdumont** [comment]
Here is the updated model with updated employment categories and the combination of the activity variables into a single coefficient. 

```
Model run at                     : 2020-11-30 09:09:10
Estimation method                : bfgs
Model diagnosis                  : successful convergence 
Number of individuals            : 597
Number of observations           : 4273

Number of cores used             :  1 
Model without mixing

LL(start)                        : -4694.37
LL(0)                            : -4694.37
LL(final)                        : -3365.394
Rho-square (0)                   :  0.2831 
Adj.Rho-square (0)               :  0.2776 
AIC                              :  6782.79 
BIC                              :  6948.15 
Estimated parameters             :  26
Time taken (hh:mm:ss)            :  00:00:45.87 
Iterations                       :  49 
Min abs eigenvalue of hessian    :  3e-06 

Estimates:
                                  Estimate Std.err. t.ratio(0) Rob.std.err. Rob.t.ratio(0)
asc_light                           1.9123   0.1895      10.09       0.4945           3.87
asc_medium                          0.6363   0.2033       3.13       0.5249           1.21
asc_heavy                           0.0000       NA         NA           NA             NA
beta_v1_industry_retail             0.0000       NA         NA           NA             NA
beta_v1_industry_industrial        -2.7745   0.2023     -13.72       0.6607          -4.20
beta_v1_industry_production        -1.5323   0.1900      -8.07       0.7209          -2.13
beta_v1_industry_ed_pub_other_ser   0.8714   0.3918       2.22       0.9176           0.95
beta_v1_industry_transportation    -3.0136   0.1567     -19.23       0.6575          -4.58
beta_v1_industry_info_fire_prof    -0.3348   0.3430      -0.98       1.0568          -0.32
beta_v1_industry_medical_services  11.2389 431.8909       0.03       0.6574          17.10
beta_v1_industry_leisure           11.3920 183.3128       0.06       0.6309          18.06
beta_v2_industry_retail             0.0000       NA         NA           NA             NA
beta_v2_industry_industrial        -1.0644   0.2005      -5.31       0.5787          -1.84
beta_v2_industry_production        -0.8652   0.2018      -4.29       0.6608          -1.31
beta_v2_industry_ed_pub_other_ser   1.5782   0.3956       3.99       1.3129           1.20
beta_v2_industry_transportation    -1.6657   0.1598     -10.42       0.5221          -3.19
beta_v2_industry_info_fire_prof     0.3779   0.3479       1.09       1.3558           0.28
beta_v2_industry_medical_services  14.0629 431.8908       0.03       0.7490          18.77
beta_v2_industry_leisure           10.9388 183.3132       0.06       0.7382          14.82
beta_v1_activity_deliver_pickup     0.0000       NA         NA           NA             NA
beta_v1_activity_service            3.7874   0.1807      20.96       0.5474           6.92
beta_v2_activity_deliver_pickup     0.0000       NA         NA           NA             NA
beta_v2_activity_service            2.3663   0.1795      13.18       0.5043           4.69
beta_v1_dist_00_02                  0.0000       NA         NA           NA             NA
beta_v1_dist_02_05                  0.6813   0.2277       2.99       0.3667           1.86
beta_v1_dist_05_10                  0.6196   0.1947       3.18       0.4541           1.36
beta_v1_dist_10_20                 -0.3005   0.1820      -1.65       0.3784          -0.79
beta_v1_dist_20_p                  -1.1137   0.1748      -6.37       0.3982          -2.80
beta_v2_dist_00_02                  0.0000       NA         NA           NA             NA
beta_v2_dist_02_05                  0.8594   0.2342       3.67       0.4310           1.99
beta_v2_dist_05_10                  0.9896   0.1987       4.98       0.5121           1.93
beta_v2_dist_10_20                 -0.0153   0.1885      -0.08       0.4808          -0.03
beta_v2_dist_20_p                  -0.2769   0.1777      -1.56       0.4885          -0.57

Overview of choices for model component "MNL"
                                   light  medium   heavy
Times available                  4273.00 4273.00 4273.00
Times chosen                     2112.00 1247.00  914.00
Percentage chosen overall          49.43   29.18   21.39
Percentage chosen when available   49.43   29.18   21.39
```

**Comment by ColSmith76** [comment]
the vehicle choice model is completed -- documentation done, and application tested and working
